CMakeLists.txt

```diff
@@ -22,9 +23,26 @@ find_package(catkin REQUIRED COMPONENTS
 
 find_package(OpenCV REQUIRED)
 
+add_message_files(
+  FILES
+  LinearPosition.msg
+  LinearVelocity.msg
+  AngularPosition.msg
+  AngularVelocity.msg
+)
+
+
+
+generate_messages(
+  DEPENDENCIES
+  std_msgs
+  mavros_msgs
+  sensor_msgs
+)
+
 catkin_package(
   INCLUDE_DIRS src
-  CATKIN_DEPENDS roscpp rospy std_msgs mavros mavros_msgs cv_bridge image_transport sensor_msgs
+  CATKIN_DEPENDS message_runtime roscpp rospy std_msgs mavros mavros_msgs cv_bridge image_transport sensor_msgs
```

Comment:
I don't know if it matters that much. We can move message_runtime and start with roscpp. just to follow the convention.

Reply:
Okay. I'll remove it